paasta_tools/tron_tools.py

```diff
@@ -1169,15 +1169,48 @@ def validate_complete_config(
     return []
 
 
-def get_tron_namespaces(cluster, soa_dir):
+def _is_action_spark(action_info):
+    if action_info.get("executor", "") == "ssh" and "spark-submit" in action_info.get(
+        "command", ""
+    ):
+        return True
+    return False
+
+
+def get_tron_namespaces(cluster, soa_dir, k8s_namespace="tron"):
```

Comment:
nit: pretty sure this is what the current types look like (which i think the updated code will violate since the spark code will return `Set[str]`
```suggestion
def get_tron_namespaces(cluster: str, soa_dir: str, k8s_namespace="tron") -> List[str]:
```